specification/src/main/asciidoc/platform/ResourcesNamingInjection.adoc

```diff
@@ -417,8 +419,11 @@ resource injection, as well as PostConstruct and PreDestroy callbacks.
 In addition, if CDI is enabled—which it is by default—these classes also
 support CDI injection, as described in
 <<a2112, Support for Dependency
```

Comment:
elsewhere you've been removing the <<a2112, syntax, but it doesn't appear to be fully removed here. Have I misunderstood the change, or is this one needing a little tidying up?

Reply:
Nope, this is the normal syntax for references.  Look for [[a2112]] for the link.

The other ones that I changed were place holders that I did previously as a reminder for the footnote work that needed to be done.